spec/suites/GeometryUtilSpec.js

```diff
@@ -10,28 +10,111 @@ describe("L.GeometryUtil", function () {
 	});
 
 	describe("readableDistance", function () {
-		it("metric", function () {
-			expect(L.GeometryUtil.readableDistance(1000, true)).to.eql('1000 m');
-			expect(L.GeometryUtil.readableDistance(1500, true)).to.eql('1.50 km');
-			expect(L.GeometryUtil.readableDistance(1500, 'metric')).to.eql('1.50 km');
+		describe("metric", function () {
+			it("returns meters or kilometers", function() {
+				expect(L.GeometryUtil.readableDistance(1000, true)).to.eql('1000 m');
+				expect(L.GeometryUtil.readableDistance(1500, true)).to.eql('1.50 km');
```

Comment:
Ja snap ik. Maar bij geld begrijp ik het bv. 1 Euro en 20 cent is 1,20 Euro. Daar hebben die laatste nullen nut.

Maar bij lengte ben ik geen 1,8800000 Meter. En een marathon is geen 42,195000000 kilometer).